core/storage/skill/gae_models.py

```diff
@@ -254,27 +254,26 @@ def get_user_id_migration_policy():
         return base_models.USER_ID_MIGRATION_POLICY.NOT_APPLICABLE
 
     @classmethod
-    def fetch_page(cls, page_size, cursor, sort_by):
+    def fetch_page(cls, page_size, urlsafe_start_cursor, sort_by):
         """Returns the models according to values specified.
 
         Args:
-            page_size: number. Number of skills to fetch
-            cursor: str. The cursor to the next page.
+            page_size: int. Number of skills to fetch.
+            urlsafe_start_cursor: str. The cursor to the next page.
             sort_by: str. A string indicating how to sort the result.
 
         Returns:
-            4-tuple(query_models, next_cursor, more, total_count). where:
+            3-tuple(query_models, urlsafe_start_cursor, more). where:
                 query_models: list(AugmentedSkillSummary). The list of summaries
```

Comment:
Hmm. Just to check -- in other docstrings, you just had augmented_skill_summary. Can you verify whether those should be lists instead?

Reply:
I have used list(AugmentedSkillSummary).